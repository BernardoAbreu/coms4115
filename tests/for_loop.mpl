array_real = [6.12, 3.1415, 2.21, 9.0001, 123.0, 5.6];

for i in array_real {
	print(str(i) + "\n");
}