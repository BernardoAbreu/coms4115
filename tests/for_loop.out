6.12
3.1415
2.21
9.0001
123
5.6
